src/model/draft/mef-legato-interfaces.yang

```diff
@@ -495,9 +502,12 @@ module mef-legato-interfaces {
                    Envelopes.";
                 leaf envelope-id {
                   type leafref {
-                    path "../../ingress-envelopes/envelope/id";
```

Comment:
This leafref previously refered to envelopes on this UNI, but now refers to envelopes on all UNIs, which is wrong.  All the similar changes you have made have this problem.  Either go back to relative paths, or limit the UNI to the current UNI.

Reply:
Ok. Let me check.